EIPS/eip-4973.md

```diff
@@ -0,0 +1,102 @@
+  /// Note: For a reliable `_from` parameter, retrieve the transaction's
+  /// authenticated `from` field.
+  event Attest(address indexed _to, uint256 indexed _tokenId);
+  /// @dev This emits when an existing ABT is revoked from an account and
+  /// destroyed by any mechanism.
+  /// Note: For a reliable `_from` parameter, retrieve the transaction's
+  /// authenticated `from` field.
+  event Revoke(address indexed _to, uint256 indexed _tokenId);
+  /// @notice Find the address bound to an ERC4973 account-bound token
+  /// @dev ABTs assigned to zero address are considered invalid, and queries
+  ///  about them do throw.
+  /// @param _tokenId The identifier for an ABT
+  /// @return The address of the owner bound to the ABT
+  function ownerOf(uint256 _tokenId) external view returns (address);
+}
+```
+
+See [`EIP-721`](./eip-721.md) for a definition of its metadata JSON Schema.
+
+## Rationale
+
+### Interface
+
+`EIP-4973` shall be maximally backward-compatible but still only expose a minimal and simple to implement interface definition.
+
+As [`EIP-721`](./eip-721.md) tokens have seen widespread adoption with wallet providers and marketplaces, using its `ERC721Metadata` interface with [`EIP-165`](./eip-165.md) for feature-detection potentially allows implementers to support `EIP-4973` tokens out of the box.
+
+If an implementer of [`EIP-721`](./eip-721.md) properly built [`EIP-165`](./eip-165.md)'s `function supportsInterface(bytes4 interfaceID)` function, already by recognizing that [`EIP-721`](./eip-721.md)'s track and transfer interface component with the identifier `0x80ac58cd` is not implemented, transferring of a token should not be suggested as a user interface option.
+
+Still, however, since `EIP-4973` supports [`EIP-721`](./eip-721.md)'s `ERC721Metadata` extension, an account-bound token should be displayed in wallets and marketplaces with no changes needed.
+
+Although other possible implementations of account-bound tokens are possible, e.g., by having all transfer functions revert, `EIP-4973` is superior as it supports feature detection through [`EIP-165`](./eip-165.md).
+
+### Exception handling
+
+Given the non-transferable between accounts property of ABTs, if a user's keys to an account or a contract get compromised or rotated, a user may lose the ability to associate themselves with the token. In some cases, this can be the desired effect. Therefore, it is suggested to implement re-issuance and potential revocation processes on the issuer side to enable recourse when it is not.
+
+This document is deliberately abstaining from offering a standardized form of exception handling in cases where user keys are compromised or rotated.
+
+In cases where implementers want to make account-bound tokens sharable among different accounts, e.g., to avoid losing access when keys get compromised, we suggest issuing the account-bound token towards a contract's account that implements a multi-signature functionality.
```

Comment:
```suggestion
In cases where implementers want to make account-bound tokens shareable among different accounts, e.g., to avoid losing access when keys get compromised, we suggest issuing the account-bound token towards a contract's account that implements a multi-signature functionality.
```